File: web-deteccion/src/app/royal/modulos/framework-comun/core/dominio/filtro/FiltroComunTipopago.ts
import { DominioPaginacion } from 'src/app/royal/framework/modelo/generico/DominioPaginacion';
 

export class FiltroComunTipopago {
    constructor(){
        this.paginacion = new DominioPaginacion();
    }
    
    paginacion: DominioPaginacion;
    
    tipopago : string;
    descripcion : string;
    restringidoflag : string;
    cartaflag : string;
    generarnumeracionflag : string;
    entregableflag : string;
    ultimousuario : string;
    ultimafechamodif : Date;
    timestamp : object;
    estado : string;
    vouchersumarizadoflag : string;
    disponiblefeederflag : string;
    descripcioningles : string;
    codigosiaf : string;
    codigofiscal : string;
    tipopagortps : string;
    tipoexportar:string
}

File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/core/dominio/filtro/FiltroComunTipopago.java
package net.royal.spring.core.dominio.filtro;

import net.royal.spring.core.dominio.BeanTipopago;

import java.util.Date;
import javax.persistence.Column;
import javax.persistence.EmbeddedId;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.persistence.Transient;
import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Pattern;
import javax.validation.constraints.Size;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;
import net.royal.spring.framework.modelo.generico.DominioTransaccion;
import net.royal.spring.framework.web.rest.UDeserializers;
import net.royal.spring.framework.web.rest.USerializers;

/**
 * 
 * 
 * @tabla dbo.TipoPago
*/
public class FiltroComunTipopago extends DominioTransaccion implements java.io.Serializable{


	private String tipopago;
	
	private String descripcion;
	private String estado;
	private String tipoexportar;
	
	
	
	public String getDescripcion() {
		return descripcion;
	}
	public void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}
	public String getEstado() {
		return estado;
	}
	public void setEstado(String estado) {
		this.estado = estado;
	}
	public String getTipoexportar() {
		return tipoexportar;
	}
	public void setTipoexportar(String tipoexportar) {
		this.tipoexportar = tipoexportar;
	}
	/**
	 * 
	 * 
	 * @campo TipoPago
	*/
	public String getTipopago() {
		return tipopago;
	}

	/**
	 * 
	 * 
	 * @campo TipoPago
	*/
	public void setTipopago(String tipopago) {
		this.tipopago = tipopago;
	}

	private DominioPaginacion paginacion;

	public DominioPaginacion getPaginacion() {
		if (paginacion == null)
			paginacion = new DominioPaginacion();
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}


}
